natscontext/context.go

```diff
@@ -262,12 +262,21 @@ func KnownContexts() []string {
 
 // SelectedContext returns the name of the current selected context, empty when non is selected
 func SelectedContext() string {
+	return readCtxFromFile("context.txt")
+}
+
+// OldCtx returns the name of the previous selected context, empty if it hasn't been selected before
+func OldCtx() string {
```

Comment:
`PreviousContext` maybe better name and more in line with like `SelectedContext`, also name it `previous-context.txt`